wifi_password/wifi_password.py

```diff
@@ -34,7 +34,7 @@ def get_ssid():
 
     elif sys.platform == "linux":
         if which("iwgetid") is None:
-            print_error("Can't find the 'iwgetid' command")
+            print_error("Can't find the 'iwgetid' command\nPlease ensure wifi-tools is installed on your machine.")
```

Comment:
Is that supposed to say `wifi-tools`? Or did you mean `wireless-tools`?

Reply:
Nope you are correct, I even double checked everything and missed it. it is wireless-tools sorry. One second